script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5.txt

```diff
@@ -112,15 +112,15 @@ the corpses move towards the firing officers―――
 I feel a white flame burning in my throat.
 I didn't even have the chance to look away.
 The sense of duty and urgency I had while running towards the city have been wiped clean.
-Right now, the only thing left in my mind is pure awe.
+Right now, the only thing left in my mind is pure awe.//to be rechecked
 <Page33>
 That's a vampire's true nature.
 A thing that simply kills people just by existing.
 And now it is an enemy that we must defeat no matter what―――
 <Page34>
 "...There's no sign of them. Those guys from the church.
 If they really were so adamant about hunting vampires, they would have been here by now... I guess they're not in this country after all.
-But that doesn't explain the unusually strict control of traffic... Perhaps the Executors are in plain uniform instead of their usual battle gear."
+But that doesn't explain the unusually strict control of traffic... Perhaps the Executors are in plain uniform instead of their usual battle gear."//to be rechecked
```

Comment:
"plainclothes" is the word I meant to use.
What was the recheck on this line?

Reply:
it has something to do with the battle gear iirc

Reply:
I think Petrikow's comment was something about this being a jab at the church employees being back office types but I don't think we got a clear idea of what to change it to

Reply:
I mean tbh, I think after revisiting this jp it's just like a general mistranslation.
> 　でも、その割にはきっちり交通規制はしているし……ここの代行者、実戦派じゃなくて制服組なのかしら」

"battle gear" is coming from 実戦派 which is nothing like battle gear, it's more like "paramilitary wing" or something. Petrikow is definitely right. This is like
> Perhaps the executioners they sent are just some pencil-pushers rather than real agents

or something. Not sure of a good word for it atm, so I'm just borrowing "agent".

Reply:
operative works as an alternative :)